src/Auth.ts

```diff
@@ -706,6 +718,24 @@ export class Auth {
     });
   }
 
+  private async ensureAccessTokenWithAccessToken(resource: string, logger: Logger, debug: boolean): Promise<AccessToken | null> {
```

Comment:
Let's pick a different name, this sounds odd 🤣

Reply:
Right :-) Suggestions? How about `ensureAccessToken` or `returnAccessToken`,